src/ui/component/syncPassword/view.jsx

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import { Form, FormField } from 'component/common/form';
+import Button from 'component/button';
+import Card from 'component/common/card';
+import { setSavedPassword } from 'util/saved-passwords';
+
+type Props = {
+  getSync: (?string) => void,
+  getSyncIsPending: boolean,
+};
+
+function SyncPassword(props: Props) {
+  const { getSync, getSyncIsPending } = props;
+  const [password, setPassword] = React.useState('');
+  const [rememberPassword, setRememberPassword] = React.useState(true);
+
+  function handleSubmit() {
+    if (rememberPassword) {
+      setSavedPassword(password);
+    }
+
+    getSync(password);
+  }
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <Card
+        title={__('Enter Your LBRY Password')}
```

Comment:
Probably could use some help text here? This is when you are logging in to an account that was encrypted with a password. You need to use the password you used on the previous device

Reply:
"Wallet Password [?]"

[?]: Your wallet password was set when you previously installed LBRY on another device. 

where [?] = mouseoverable help thing

Does it have to be another device though? Is wallet password put into OS keyring? If not, couldn't it be the same machine/install?

Reply:
Good point, it could be saved from an old install on the same device.

Reply:
Let's just say the following, it's nice and ambiguous:

[?]: You set your wallet password when you previously installed LBRY.